mozartdata/transforms/dev_reporting/goodr_com_usage_kpis.sql

```diff
@@ -0,0 +1,145 @@
+/*
+    This table shows the daily KPIs for goodr.com usage.
+    Columns:
+        date:
+            Date reported on. This comes from dim.date to prevent issues
+            with a date not existing in Shopify.
+        sessions:
+            Total sessions as defined by Shopify for each day.
+        users:
+            Labeled "Online store visitors" in Shopify, this is the number of
+            unique users that visited the site wach day regardless of their
+            session count.
+        total_customers:
+            This is the total number of users that bought a product in a day.
+    Todos:
+        - handle leap years
+ */
+
+with
+    marketing_totals as (
+                            select
+                                date::date           as date
+                              , round(sum(spend), 2) as total_spend
+                              , sum(impressions)     as total_impressions
+                            from
+                                goodr_reporting.performance_media as pm
+                            where
+                                  lower(pm.account_country) = 'usa'
+                              and date >= '2024-01-01'
+                              and date <= current_date
+                            group by
+                                date
+                        )
+  , revenue_totals as (
+                            select
+                                gl_tran.transaction_date::date    as date
+                              , round(sum(gl_tran.net_amount), 2) as total_revenue
+                            from
+                                fact.gl_transaction as gl_tran
+                            where
+                                  gl_tran.posting_flag = true
+                              and lower(gl_tran.channel) = 'goodr.com'
```

Comment:
Again, we know the values here. We don't need to lower case this. Just look for Goodr.com
```suggestion
                              and gl_tran.channel = 'Goodr.com'
```

Reply:
I mean, its really a just in case kind of thing. But if you don't think its necessary you know better than me. I'm an overworker.